xcode/Subconscious/Shared/Components/Detail/Detail.swift

```diff
@@ -362,7 +358,7 @@ enum DetailAction: Hashable, CustomLogStringConvertible {
     case scenePhaseChange(ScenePhase)
 
     /// Wrapper for editor actions
-    case markupEditor(MarkupTextAction)
+    case markupEditor(SubtextTextAction)
```

Comment:
@bfollington d'oh I missed this one. We should rename `markupEditor` to `editor`, since the property name changed.